### DISCUSSION TITLE
Not able to access Vercel sites (Viettel - Vietnam)

### DISCUSSION THREAD

**Discussion by kieuson93** [vercel/community#2971] [Bug Report]
### Page to Investigate

vercel.com

### Steps to Reproduce

We are not able to access to Vercel home page also other sites are using vercel platform.

My ISP: Viettel ISP
Location: Vietnam

Kindly support!

Best regards,
Son

**Comment by kieuson93** [comment]


**Reply by smaeda-ks** [reply]
@kieuson93 could you share few more details?
https://gist.github.com/smaeda-ks/5997018d5fe5dce44d047299829288e1

**Reply by smaeda-ks** [reply]
@kieuson93 ok, there's something I don't really understand is going on with your local DNS server (`172.30.96.1`). It's definitely behaving buggy and you should check that. In the first dig result, the answer section contains records that should not be included there at all.

```
a.zeit-world.co.uk.     0       IN      AAAA    2620:4d:4000:6259:7:13:0:1
e.zeit-world.com.       0       IN      AAAA    2620:4d:4000:6259:7:13:0:1
b.zeit-world.org.       0       IN      AAAA    2a00:edc0:6259:7:13::2
a.zeit-world.co.uk.     0       IN      A       198.51.44.13
e.zeit-world.com.       0       IN      A       198.51.44.13
f.zeit-world.net.       0       IN      A       198.51.45.13
b.zeit-world.org.       0       IN      A       198.51.45.13
```

These are our nameservers and `ANSWER SECTION` should only contains `A` records for `custom-api-node.vercel.app`.
As a result of this, your following `curl` command is trying to send the request to those nameserver IPs rather than the actual IPs of your deployment `custom-api-node.vercel.app`, and immediately failing because port 443 isn't open on those IPs.

**Reply by kieuson93** [reply]
@smaeda-ks thanks for your investigation. But there's one thing more important that is I can't access to any sites hosted in vercel.

**Reply by smaeda-ks** [reply]
if you change your DNS server to `8.8.8.8` on your machine and everything works then this isn't a typical ISP blocking issue

**Reply by smaeda-ks** [reply]
maybe you can also run `dig +trace custom-api-node.vercel.app` and see what it shows

**Reply by kieuson93** [reply]
yes I'm currently using 8.8.8.8. Not working at all. It happens back today. Few months ago, I got same issue.  @smaeda-ks

**Reply by smaeda-ks** [reply]
@kieuson93 that doesn't look like a complete response and it's still using `172.30.96.1`. Try:
`$ dig +trace custom-api-node.vercel.app @8.8.8.8`

**Reply by smaeda-ks** [reply]
`$ dig custom-api-node.vercel.app @ns-tld1.charlestonroadregistry.com.`

**Reply by smaeda-ks** [reply]
shoot me a DM https://twitter.com/jrsyo we can hop on a quick call

**Comment by agamm** [comment]
Also for Thailand (with cloudflare as the NS server).

**Reply by smaeda-ks** [reply]
please collect and share the result of https://gist.github.com/smaeda-ks/5997018d5fe5dce44d047299829288e1

in "text" if possible
